src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml

```diff
@@ -42,8 +41,8 @@
                                 </div>
                             </div>
 
-                            <div data-test-id="forgot-password-link" class="bc-c-data-link bc-c-login-page-link">
-                                <a asp-action="ResetPassword" asp-route-returnurl="@ViewData["ReturnUrl"]" target="_blank">
+                            <div class="bc-c-data-link bc-c-login-page-link">
```

Comment:
does that class exist? `bc-c-login-page-link`

Reply:
I was trying to switch these over to the NHS ones also, but I can't find any that look correct, so I'm going to stick with the bc-c-login-page-link here I think